Hospital.Domain/Entities/Patient.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hospital.Domain.Entities;
+/// <summary>
+/// Represents a patient registered in the clinic.
+/// </summary>
+public class Patient
+{
+    /// <summary>
+    /// Primary key (auto-generated by database).
+    /// </summary>
+    public int Id { get; set; }
+
+    /// <summary>
+    /// Passport number of the patient.
+    /// </summary>
+    public string? Passport { get; set; }
+
+    /// <summary>
+    /// Full name of the patient.
+    /// </summary>
+    public required string FullName { get; set; }
+
+    /// <summary>
+    /// Gender of the patient.
+    /// </summary>
+    public required Gender Gender { get; set; }
+
+    /// <summary>
+    /// Birth date of the patient.
+    /// </summary>
+    public DateOnly BirthDate { get; set; }
+
+    /// <summary>
+    /// Residential address.
+    /// </summary>
+    public string? Address { get; set; }
+
+    /// <summary>
+    /// Blood group of the patient.
+    /// </summary>
+    public BloodGroup BloodGroup { get; set; }
+
+    /// <summary>
+    /// Rhesus factor of the patient.
+    /// </summary>
+    public RhesusFactor Rhesus { get; set; }
+
+    /// <summary>
+    /// Contact phone number.
+    /// </summary>
+    public string? Phone { get; set; }
+
+    /// <summary>
+    /// Collection of appointments for this patient.
+    /// </summary>
+    public ICollection<Appointment> Appointments { get; set; } = new List<Appointment>();
```

Comment:
Инициализацию списка можно упростить 
```suggestion
    public ICollection<Appointment> Appointments { get; set; } = [];
```